### ISSUE TITLE
ticketing system for email to info@, security@, etc.

### ISSUE THREAD

**Issue by pdurbin** [opendp/opendp#241]
As discussed this morning, I'm proposing that OpenDP use the IQSS/HMDC ticketing system at https://help.hmdc.harvard.edu for tracking correspondence that shouldn't be made public. (Dataverse uses this system as well.) Because it runs software called Request Tracker, it's often referred to as "RT".

Without a ticketing system, it's hard to know who has replied to an inquiry, what was said, if it's resolved or not, etc.

The obvious email to convert to a ticketing system is info@opendp.org. (Currently, info@opendp.org forwards to a group under g.harvard.edu.) We would get in touch with the IQSS/HMDC RT team via help@iq.harvard.edu and ask them to set up a "queue" for OpenDP.

While we're at it, I would suggest that now is a good time to set up the email address security@opendp.org so that security researchers and others can contact us privately about any security concerns. For now it's probably fine to put both "info" and "security" messages in the same RT queue but if anything thinks starting off with two queues is better, please say so.

Finally, I'll mention that while I'm focused on info@opendp.org and a new address called security@opendp.org, this spreadshet contains many more email addresses for OpenDP that may or may not be in regular use: https://docs.google.com/spreadsheets/d/1YnOtvQkql4880kk2jhFXlW5isj3NXcqiLEs90RAJ8Is/edit?usp=sharing . For now, I'd consider other email addresses out of scope.

If this plan sounds reasonable, I just need the go ahead to get the ball rolling. I'll probably need help with changing the forwarding for info@opendp.org.

**Comment by anniewu332** [comment]
Sounds good to me.  Please proceed. Thank you, Phil.

**Comment by pdurbin** [comment]
Ok! I started the conversation in https://help.hmdc.harvard.edu/Ticket/Display.html?id=307587

By the way, I mentioned this the other day but this ticketing system will be Harvard/HMDC branded in the sense that when people email us, they will get a link like that "help.hmdc" link above. In practice, this hasn't been a problem for Dataverse. People seem to recognize that Harvard is at least partially behind these software projects.

**Comment by pdurbin** [comment]
Here's a sneak peak of the new ticketing system we're setting up for OpenDP:

![Screen Shot 2021-08-31 at 9 46 25 AM](https://user-images.githubusercontent.com/21006/131513933-9eccf6e1-05f0-4101-b561-cdc54a4dafad.png)

Here's a link to the test ticket above: https://help.hmdc.harvard.edu/Ticket/Display.html?id=308114

As discussed above, we're planning to use a single queue called "opendp" for at least a couple opendp email address. opendp@help.hmdc.harvard.edu is what you could think of as the "native" email address and should be operational now. When you email that address, a ticket should be created and notifications should go (for now) to me, @anniewu332 and @raprasad.

IQSS/HMDC is still working on getting tickets created to work from opendp email accounts. As part of this, I'll set up security@opendp.org.

Here are things we can do:

- Play around with the queue (for Annie, Raman, and Phil).
- Decide who should get notifications of new tickets. (Their RT accounts will become "privileged" if they aren't already.)
- Perhaps a bit of training on how to use the system (I'm happy to do this).
- Get prepared to switch info@opendp.org to the ticketing system.

**Comment by pdurbin** [comment]
Just a quick update to say that I added the various DNS entries below as part of the setup.

![image](https://user-images.githubusercontent.com/21006/132704964-487e0312-0aaf-4d83-aa95-f7a449a3bd57.png)

**Comment by pdurbin** [comment]
I created security@opendp.org and set up forwarding (below). It seems to work fine.

<img width="866" alt="Screen Shot 2021-09-09 at 11 03 00 AM" src="https://user-images.githubusercontent.com/21006/132714013-0d45d038-ad78-4228-9e58-4660036172f1.png">

It sounds like we don't need the Mailgun DNS entries after all so I'll delete them.

After a little more testing we should be ready to switch info@opendp.org from a Google Group to RT.

**Comment by pdurbin** [comment]
I configured info@opendp.org (like security@opendp.org) to forward into RT. It all seems to be working fine.

The last thing I could think to set up was some sort of an autoreply so that people who write know what their ticket number is. The reply looks like this:

![Screen Shot 2021-09-09 at 1 51 04 PM](https://user-images.githubusercontent.com/21006/132737262-6f7235ef-1c38-4983-8f01-860a8092df61.png)

As I said above, I'm happy to do a little training on using RT for anyone who wants it. In practice, I imagine @raprasad @anniewu332 and I will keep an eye on incoming messages and grab other people to help as needed.

Closing.
